.bazelrc

```diff
@@ -11,3 +11,8 @@ import %workspace%/ci.bazelrc
 
 # Try to import a local.rc file; typically, written by CI
 try-import %workspace%/local.bazelrc
+
+# The CNIOBoringSSL library uses C++14 features like 'enable_if_t' macro support.
```

Comment:
Curious. Which dependency is bringing in CNIOBoringSSL?

Reply:
This is used by swift_grpc_library from rules_swift here:
https://github.com/bazelbuild/rules_swift/blob/master/third_party/com_github_apple_swift_nio_ssl/BUILD.overlay#L39
Previously you didn't have any swift_grpc_library or swift_proto_library examples so it wasn't being evaluated.